read-fonts/src/array.rs

```diff
@@ -147,6 +147,17 @@ impl<'a, T: FontRead<'a> + VarSize> VarLenArray<'a, T> {
             Some(next)
         })
     }
+
+    pub fn counted_size(&self, elements: usize) -> Result<usize, ReadError> {
```

Comment:
I think I would have to check the code to know what a counted size is. Suggest at least adding a comment, potentially renaming, for example element_size or size_of_elements would be more obvious.

I immediately wonder if one would ever want the size of something other than 0..elements, e.g. should we take a range rather than just #elements. Admittedly I don't have a specific use in mind so feel free to ignore.